app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
```

Comment:
It's not clear what the `mv` command should look like. It would be helpful to add a comment or an example of expected command. (Code Efficiency issue)